docs/services-guide.md

```diff
@@ -4,772 +4,133 @@ toc: true
+This guide is an overview of enabling cloud service 
+provisioning on an existing Kubernetes target cluster, 
+including how to integrate Crossplane with existing cloud networking configurations 
+to provide secure managed service connectivity.  Step-by-step instructions are 
+provided for [GCP][gcp-services-guide], [AWS][aws-services-guide], 
+and [Azure][azure-services-guide].
+
+To dynamically provision a new Kubernetes target cluster 
+see the Stacks Guides for [GCP][stack-guide-gcp], 
+[AWS][stack-guide-aws], and [Azure][stack-guide-azure].
+
+## Table of Contents
+1. [Introduction](#introduction)
+1. [Secure network connectivity for cloud services](#secure-network-connectivity-for-cloud-services)
+1. [Dynamic provisioning with claims and classes](#dynamic-provisioning-with-claims-and-classes)
+1. [Connection secret for pods in a deployment](#connection-secret-for-pods-in-a-deployment)
+1. [Next Steps](#next-steps)
+1. [Learn More](#learn-more)
+
+## Introduction
+Cloud service provisioning can be added to existing clusters by installing 
+Crossplane directly onto the target cluster. Crossplane is designed to 
+integrate with existing cloud networking and security resources, so 
+managed services like RDS, CloudSQL, and Azure DB can be provisioned 
+using Kubernetes objects securly consumed by pods in a cluster. 
+
+Crossplane achieves this by:
+1. establishing secure network connectivity between the worker nodes in a cluster
+and the cloud services using a set of networking and security resources
+1. populating Kuberentes `Secrets` that pods in a `Deployment` can use to 
+securely access the managed service
+
+## Secure network connectivity for cloud services
+Crossplane currently supports private IP secure connectivity for
+AWS, GCP, and Azure Stacks. Managed services instances
+are made available on the cluster's prviate network(s) so pods can access them.
+Crossplane also supports configuring ingress/egress rules to further restrict 
+allowed network traffic.
+
+While each cloud provider uses different resources for establishing 
+secure connectivity between a Kubernetes cluster (EKS, GKE, AKS) 
+and managed services (RDS, CloudSQL, and Azure DB), the basic pattern is the
+same:
+1. Configure cluster networking
+   * network(s) and subnet(s) - L3 networking for the worker nodes
+1. Enable managed service access: 
+   * private service connection / endpoint - make services available via peering or other
+   * private IP range(s) or subnet group - the private IPs a managed service will get
+   * security groups or network rules - to restrict network traffic
+1. Provision a managed service instance
+   * creates an instance (e.g. RDS, CloudSQL, Azure DB MySQL instance)
+   * assigns a private IP from the private IP range above
+1. Securely use the managed service with secrets
+   * pods on a cluster node can access the managed service via private IP
+
+Crossplane provides Kubernetes objects for all of the above, so you can
```

Comment:
We tend to say "Kubernetes resources" rather than "objects" in most places. It's hard to determine which is more correct; for example https://blog.openshift.com/kubernetes-deep-dive-api-server-part-1/ and https://stackoverflow.com/a/52314551 seem to contradict each other.